test/UnitTest/Altinn.Platform.Storage.UnitTest.csproj

```diff
@@ -26,153 +26,174 @@
-      <Link>stylecop.json</Link>
-    </AdditionalFiles>
-  </ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="8.0.3" />
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+		<PackageReference Include="Moq" Version="4.20.70" />
+		<PackageReference Include="xunit" Version="2.7.0" />
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.5.7">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="coverlet.collector" Version="6.0.2">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+	</ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\Storage\Altinn.Platform.Storage.csproj" />
-  </ItemGroup>
+	<ItemGroup Condition="'$(Configuration)'=='Debug'">
+		<PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.556">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
+		</PackageReference>
+		<AdditionalFiles Include="..\..\stylecop.json">
+			<Link>stylecop.json</Link>
+		</AdditionalFiles>
+	</ItemGroup>
 
-  <PropertyGroup>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <NoWarn>$(NoWarn);1591</NoWarn>
-  </PropertyGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\..\src\Storage\Altinn.Platform.Storage.csproj" />
+	</ItemGroup>
 
-  <ItemGroup>
-    <Compile Remove="data\apps\tdd\endring-av-navn\logic\**" />
-    <Compile Remove="data\roles\user_1 - Copy\**" />
-    <Content Remove="data\apps\tdd\endring-av-navn\logic\**" />
-    <Content Remove="data\roles\user_1 - Copy\**" />
-    <EmbeddedResource Remove="data\apps\tdd\endring-av-navn\logic\**" />
-    <EmbeddedResource Remove="data\roles\user_1 - Copy\**" />
-    <None Remove="data\apps\tdd\endring-av-navn\logic\**" />
-    <None Remove="data\roles\user_1 - Copy\**" />
-  </ItemGroup>
+	<PropertyGroup>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+		<NoWarn>$(NoWarn);1591</NoWarn>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <Content Remove="data\apps\tests-sailor.json" />
-    <Content Remove="data\m1000-instances.json" />
-    <Content Remove="data\response_deny.json" />
-    <Content Remove="data\response_multi_permit.json" />
-    <Content Remove="data\response_permit.json" />
-    <Content Remove="data\roles\user_1\party_1000\roles.json" />
-    <Content Remove="data\roles\user_1\party_1600\roles.json" />
-    <Content Remove="data\roles\user_1\party_50000000\roles.json" />
-    <Content Remove="data\roles\user_1\party_500\roles.json" />
-    <Content Remove="data\roles\user_3\party_1606\roles.json" />
-    <Content Remove="data\roles\user_3\party_1600\roles.json" />
-  </ItemGroup>
+	<ItemGroup>
```

Comment:
could this section be deleted a a whole? Don't think the files are in the repository any more